packages/core/src/lib/icon/icons.ts

```diff
@@ -1,5 +1,9 @@
 import * as MDI from '@mdi/js';
 
+/**
+ * Keys should match MDI name
+ * e.g. 'account-multiple' for MDI.mdiAccountMultiple
+ */
```

Comment:
FYI @ethanlook ... I didn't change anything as of yet but we should for the ones that do not match.

Reply:
We need to coordinate with @nathartman and design about this since the names are also used in Figma. I don't feel strongly that the names match the MDI name.